config.yaml

```
@@ -288,7 +288,16 @@ options:
       profile will be extracted from '/var/log/cloud-init-output.log', fallback is 'level1_server'.
       Options are: '' (disable profile check), 'level1_server', 'level2_server',
                    'level1_workstation' or 'level2_workstation'
+      This option cannot be used with cis_audit_tailoringfile at the same time.
       See also https://ubuntu.com/security/certifications/docs/cis-audit
+  cis_audit_tailoringfile:
+    default: ""
+    type: string
+    description: |
+      Custom cis profile xml file. The content of the file will be placed at
```

Make it clear that this accepts content, not a filename.

```suggestion
      Custom cis profile xml file content. The content will be saved in
```